app/styles/ui/_banners.scss

```diff
@@ -74,4 +75,11 @@
       transition: height 225ms ease-in-out 175ms, opacity 175ms ease-in;
     }
   }
+
+  // When zooming, give the banner more space to display.
+  @media (max-width: 960px) {
+    height: auto;
+    padding-top: var(--spacing-half);
+    padding-bottom: var(--spacing-half);
```

Comment:
I noticed that when I first tried to make message which was really long, the banner didn't handle text wrapping very well so I figured that meant zooming didn't wrap. This css just makes it so that when zoomed, there is not height constraint so the banner will always be tall enough for the text. A concern would be if the banner got so tall it caused content below to be not visible, but this shouldn't happen as with the 30px in non-zoomed mode we are limiting the banner content to a one liner which wraps to three in the worst case.

Before:
![Showing silicon machine message being clipped at 200% zoom](https://github.com/desktop/desktop/assets/75402236/ff0d4375-5e0c-4cd1-8208-0a3591abb197)

After:
![Showing silicon machine message taking the space required to show all three lines of text](https://github.com/desktop/desktop/assets/75402236/ba2657d8-da6e-485f-862a-2cda506ef5de)